frontend/packages/console-plugin-sdk/src/typings/dashboards.ts

```diff
@@ -0,0 +1,38 @@
+import { Extension } from '.';
+import { GetHealthStateFunction } from '@console/internal/components/dashboards-page/overview-dashboard/health-card';
+
+namespace ExtensionProperties {
+  export interface OverviewHealthURLSubsystem {
+    title: string;
+    url: string;
+    responseHandler?: any;
+    fetchMethod?: any;
```

Comment:
suggestion: wire `fetchMethod` with `healthHandler` via type parameter, also we don't need `responseHandler` (can be part of `fetchMethod`)